### ISSUE TITLE
1.58 Power Issues (FluxNetworks, RFTools, ModularMachinery)

### ISSUE THREAD

**Issue by Mieskoh** [NillerMedDild/Enigmatica2Expert#973]
**Modpack version**
1.58

**Can the issue be reproduced?**
unknown

**Logs**
Input pastebin link to crash-report or latest log, if applicable

**Issue**
After updating the pack today, I noticed that my base didn't have any power pretty quicky.
As I went around trying to figure out the issue, I noticed that all of my nodes stopped transferring power, even with a few billion RF in their network.

I tried deleting my networks and creating a new one, which did work for a few nodes but stopped sending any power once more while I was updating all my nodes.

After rolling back to the previous version of the pack, my networks started working again.

**Comment by Darkstone158** [comment]
yeah I had this exact same issue

**Comment by NillerMedDild** [comment]
Well that's concerning, I'll look into it.

**Comment by NillerMedDild** [comment]
I didn't update Flux Networks or even make any changes to it, so I don't currently know what's causing this issue. Was/is your Flux Network connected to RFTools Powercells?

**Comment by Mieskoh** [comment]
There are some powercells that are acting as buffers for some nodes but my main storage is a mekanism matrix.

**Comment by NillerMedDild** [comment]
I think they may be the issue. @Darkstone158 do you also have Powercells attached to your Network?

**Comment by Mieskoh** [comment]
I'll try cloning the pack and removing the powercells then.
As I might be worth saying, I had other nodes powering other buffers/blocks but they did not receive power.
The strangest thing was that my ME system had 0 RF but still worked after saying that it did not have enough power.

**Comment by Darkstone158** [comment]
no i dont have any powercell attached to my system

**Comment by NillerMedDild** [comment]
Dammit.

**Comment by Mieskoh** [comment]
After testing in the duplicated instance I believe the issue I had did come from powercells.
Upon loading the world I noticed that my base was getting power, but this time my network in charge of the power input was the one broken.
I went around to see if I had any powercells connected to this network and broke the two connected to my advanced generators.
The network started to work once more and still does so far.

I had some powercells acting as buffers on the output network and it still worked, but just in case I replaced them with ones from thermal.

**Comment by NillerMedDild** [comment]
Awesome!

**Comment by Gepett0** [comment]
Just noticed the same issue with my flux network.
Although I don't have any powercells anymore I used a scanner from scannable to find the culprit but without success. So I remade the entire channel, which stopped while I was remaking it.
Issue for me was:

I have 8 dedicated Adv. Carpenter inside a compact machine, one for each needed liquid.
As soon as I connect any of the AC directly the network stops working. Breaking the flux point starts the network again. Tried preconfigured and newly crafted flux points.
Breaking and replacing of the Input Hatch fixes this.

The Adv. Thermionic seems fine with direct input, my Adv. Metallurgic uses a Capacitor Bank as buffer, and works fine too.

Just restarted the server to test rworld reloading behaviour the channel is still working.

So it could be a problem from Modular Machinery as some energy things were changed within the latest release.

TLDR: If your Flux Network refuses to work, try breaking and replacing the input hatches of MM.

**Comment by Krapht** [comment]
A bit late to the party, but I had the exact same issue with flux network refusing to transfer power.

Can't confirm anything, but I think it was related to a powercell change. After update I had more power in my cells than the network would fit. Somehow this affected flux. Once the cell network was drained of power everything has gone back to normal

**Comment by pjbardo** [comment]
> TLDR: If your Flux Network refuses to work, try breaking and replacing the input hatches of MM.

This.

Replaced my Flux Points attached to Modular Machinery with Powercells and all my problems went away.

**Comment by MaxNeedsSnacks** [comment]
Are there still any issues with Flux Networks now after you've replaced your input hatches and such?
